dnsmasq_interface.c

```diff
@@ -193,7 +193,7 @@ void FTL_new_query(unsigned int flags, char *name, struct all_addr *addr, char *
 		if(match_regex(domain) && !in_whitelist(domain))
 		{
 			// We have to block this domain if not already done
-			if(debug) logg("Blocking %s due to RegEx match", domain);
+			if(debug) logg("Blocking %s due to Regex match", domain);
```

Comment:
Does this config option supersede this line?

Reply:
Yes, there is no reason to show it in FTL's standard debug mode any more as it has proven to work exactly as expected even for large lists.

Reply:
Should this line then be removed?